src/components/Person/Person.jsx

```diff
@@ -2,0 +1,19 @@
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    <p className="Person__age">
+      {person.age ? <span>I am ${person.age}</span> : null}
+    </p>
+    {person.isMarried === true ? (
+      <p className="Person__partner">
+        {person.sex === 'm' ? (
+          <span>${person.partnerName} is my wife</span>
```

Comment:
The template literal syntax is incorrect here. Change `${person.partnerName}` to `{person.partnerName}` to properly render the partner's name.